### PR TITLE
182/Integrate Sentry Logger Service.

### PR DESCRIPTION
**Pull request by Ritu-Innostax**
Resolves: #182

Description:

Steps to test Sentry integration:-
1.Create an account on [Sentry.io](http://sentry.io/) and generate a DSN url.
2.Paste the DSN url in sentry.js file.
3.Run app and make a GET request on the users/logger/test.
4. Check the sentry dashboard, a log will be created.

### CODE REVIEW

**Review by manjeetbeniwalinnostax — APPROVED** [review]

**Comment by manjeetbeniwalinnostax** [comment]
will resolve the comments in the follow up PR

File: src/templates/node-js/controllers/controller.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<% if (isSentry) { %>
+  const Sentry = require('../utils/logger')
+  <% } %>
+
+  <% if (isWinston) { %>
+    const logger = require('../utils/logger')
+    <% } %>
```
Comment (Chaman-Raghav-Inno) [inline src/templates/node-js/controllers/controller.js:7]: formatting can be done.
please take care of formatting the codebase.

File: src/templates/node-js/controllers/controller.js [hunk 2]
```diff
@@ -23,7 +31,8 @@ const <%= defaultRoute %> = require("../models/<%- defaultRoute %>.js");
       <% } %>
       <% if(sequelizeSelected){%>
         <%= defaultRoute %>.findAll().then((<%= defaultRoute %>) => {
-          res.json(<%= defaultRoute %>);
+          if (<%= defaultRoute %>.length > 0) res.json(<%= defaultRoute %>);
+          else res.send("no user found");
```
Comment (Chaman-Raghav-Inno) [inline src/templates/node-js/controllers/controller.js:35]: this we reverted in some previous PR & now adding in it, why's that?

File: src/templates/node-js/routes/route.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ router.get('/:id',<%= defaultRoute %>.findById)
 router.patch('/:id',<%= defaultRoute %>.patch),
 router.delete('/',<%= defaultRoute %>.remove),
 router.delete('/:id',<%= defaultRoute %>.removeById)
+<% if (isSentry || isWinston) { %>
+//   /users/logger/health
```
Comment (Chaman-Raghav-Inno) [inline src/templates/node-js/routes/route.js:12]: lets remove commented code

**Review by Chaman-Raghav-Inno — COMMENTED** [review]
Approved with minor suggestion/query.

**Pull request merged**